libs/server-sdk/src/client_impl.cpp

```diff
@@ -80,7 +79,7 @@ bool EventsEnabled(Config const& config) {
     return config.Events().Enabled() && !config.Offline();
 }
 
```

Comment:
Decided to remove `NullEventProcessor` entirely from the server-side SDK, and just return a `nullptr` instead. We only need to check it in two places:
- `EventScope::Send`
- `Client::FlushAsync`